### ISSUE TITLE
Error : Event dispatcher is disposed

### ISSUE THREAD

**Issue by hugoperier** [amcharts/amcharts4#2189]
Hey, it's me again

Sometime, when I add or remove series I got the error "Event dispatcher is disposed"

I have reviewed all my code and I cant find the issue

I have update my github repository to have the runnable exemple 
https://github.com/hugoperier/amchart-problem-demo

To get the error

first add two items using the searchbar
![image](https://user-images.githubusercontent.com/26797879/75525144-963fe300-5a0f-11ea-9af3-fd1681cc9379.png)

then remove the last item on the right 
![image](https://user-images.githubusercontent.com/26797879/75525177-a8ba1c80-5a0f-11ea-92b9-ff9b34e2a289.png)

And finally click on the "10" box of the item and you can see the error

![image](https://user-images.githubusercontent.com/26797879/75525232-c4bdbe00-5a0f-11ea-93d3-379ecbaabbfa.png)

Thanks you a lot

**Comment by martynasma** [comment]
The issue is that while you remove the series, its "clone" remains in the `XYChartScrollbar`.

We will look for ways to automatically remove scrollbar series when chart series is removed/disposed.

In the interim, you can fix it by removing series from the scrollbar yourself:

```
    deleteList.forEach(element => {
      this.state.chart.series.removeIndex(element)
      this.state.chart.scrollbarX.series.removeIndex(element);
    });
```

Please note that removing series from `chart.series` automatically disposes it, so you don't have to call `dispose()` on it again.

**Comment by martynasma** [comment]
Fixed in [4.9.5](https://github.com/amcharts/amcharts4/releases/tag/4.9.5).

## [4.9.5] - 2020-03-03

### Added
- Third paramterer - `force` - added to `ChartCursor.triggerMove`. If set to `true` it will trigger all actions associated with the move even if the cursor is already at the same coordinates.

### Fixed
- Exporting from ExportMenu will now prevent cursor (and releated tooltips) from appearing just before export so they are not included in exported image.
- Panning on a `MapChart` with `panBehavior = "rotateLongLat"` was not working properly since `4.9.3`.
- Disposed series was not being properly removed from `XYChartScrollbar` if it was used there.
- Sprites with `dx`/`dy` set and `rotation != 0` were not displaying fixed-position tooltip in correct position.
- Changing `tooltipText` was causing the tooltip to hide if it was already visible and `tooltipPosition` was set to `"pointer"`.
- `XYChart` with legend positioned to right/left was not positioning `valueLabels` properly (if aligned to right).
- `TreeMap` legend was not functioning properly.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
